lib/puppet_litmus.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 require 'bolt_spec/run'
+require 'inventory'
```

Comment:
Should this be `puppet_litmus/inventory` and the filename of it `lib/puppet_litmus/inventory.rb`?